repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -184,7 +186,19 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
   }
 
   describe("readToDom") {
-    pending
+    it("should return a Document if the stream is able to be parsed") {
+      val document = filter.readToDom(new BufferedServletInputStream(new ByteArrayInputStream(samlResponseStr.getBytes(UTF_8))))
+      document should not be null
+    }
+
+    it("should throw an exception if the stream is not able to be parsed") {
+      val exception = intercept[SamlPolicyException] {
```

Comment:
So this is 100% legit Scalatest. It comes in with all of the default assertions.
http://www.scalatest.org/user_guide/using_assertions

However, I've been using the ScalaTest Matchers DSL to be consistent with our other assertions.
http://www.scalatest.org/user_guide/using_matchers#checkingForExpectedExceptions

My vote is to use the Matchers DSL, but whichever we choose to use, we should at least be consistent within this test class.